game-app/game-core/src/main/java/games/strategy/engine/data/gameparser/GameParser.java

```diff
@@ -663,7 +703,7 @@ private void parseResults(final Rule dataRule, final Production.Rule mapRule)
     List<Production.Rule.Result> ruleResults = mapRule.getRuleResults();
     if (ruleResults.isEmpty()) {
       throw new GameParseException(
-          MessageFormat.format("No results for rule {0}", dataRule.getName()));
+          MessageFormat.format("No results for production rule: {0}", dataRule.getName()));
```

Comment:
Looks like this might be either 'repair' rule or 'production rule'.
The message " for production rule" might be incorrect (?)